src/main/data/studios.json

```
@@ -2549,6 +2549,16 @@
     "code": "MSVP",
     "description": "MYSTIC STUDIOS & VINAYAK PRODUCTIONS"
   },
+  {
+    "code": "MTIM",
+    "contact": {
+      "address": "1301 E MAIN ST, BOX 58, MURFREESBORO, TN, 37132-0001, USA",
```

```suggestion
      "address": "1301 E MAIN ST, BOX 58, MURFREESBORO, TN 37132-0001, USA",
```

@japierce1028 State and zip code are on the same line in US addresses.